### ISSUE TITLE
Hunting for headings: sighted labeling vs. automatic classification of headings

### ISSUE THREAD

**Issue by Wotipati** [Wotipati/Blind_Accessibility_Papers#231]
### Links
- Paper: https://dl.acm.org/doi/10.1145/1414471.1414508

### Abstract
Proper use of headings in web pages can make navigation more efficient for blind web users by indicating semantic divisions in the page. Unfortunately, many web pages do not use proper HTML markup (h1-h6 tags) to indicate headings, instead using visual styling to create headings, thus making the distinction between headings and other page text indistinguishable to blind users. In a user study in which sighted participants labeled headings on a set of web pages, participants did not often agree on which elements on the page should be labeled as headings, suggesting why headings are not used properly on the web today. To address this problem, we have created a system called HeadingHunter that predicts whether web page text semantically functions as a heading by examining visual features of the text as rendered in a web browser. Its performance in labeling headings compares favorably with both a manually-classified set of heading examples and the combined results of the sighted labelers in our study. The resulting system illustrates a general methodology of creating simple scripts operating over visual features that can be directly included in existing tools.

- Webページに適切な見出しをつけるシステムの提案
